### PR TITLE
Wet-boew working examples #2

### PR DESCRIPTION
**Pull request by SebastianBurke**
Added wet-boew working examples:

eqht-css
filter
lightbox
stepsform
tabs
texthighlight
twitter
wb-randomize

### CODE REVIEW

File: wet-boew/eqht-css/index.json-ld
Comment (duboisp) [inline wet-boew/eqht-css/index.json-ld:1]: Merge this one with the other equal height. Either here on in your other PR.

File: wet-boew/eqht-css/index.json-ld [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+				"title": "Equal height (pure CSS)",
+				"language": "en",
+				"url": "https://wet-boew.github.io/wet-boew/docs/ref/eqht-css/eqht-css-en.html"
+			},
+			{
+				"title": "Égalisation des hauteurs (pure CSS)",
+				"language": "fr",
+				"url": "https://wet-boew.github.io/wet-boew/docs/ref/eqht-css/eqht-css-fr.html"
+			}
+		]
```
Comment (duboisp) [inline wet-boew/eqht-css/index.json-ld:44]: Let's use the docs of the other equal height and add those link under "other" group.

```suggestion
		"other": [
			{
				"title": "Documentation - Equal height (pure CSS)",
				"language": "en",
				"url": "https://wet-boew.github.io/wet-boew/docs/ref/eqht-css/eqht-css-en.html"
			},
			{
				"title": "Documentation - Égalisation des hauteurs (pure CSS)",
				"language": "fr",
				"url": "https://wet-boew.github.io/wet-boew/docs/ref/eqht-css/eqht-css-fr.html"
			}
		]
```

File: wet-boew/stepsform/index.json-ld [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		"en": "Steps Form",
+		"fr": "Formulaire à étapes"
+	},
+	"description": {
+		"en": "This component provides the ability to break a form into steps",
+		"fr": "Offre la possibilité de diviser un formulaire en différentes étapes (sans ou avec validation de formulaire)"
+	},
+	"modified": "2024-05-17",
+	"componentName": "stepsform",
+	"status": "stable",
```
Comment (duboisp) [inline wet-boew/stepsform/index.json-ld:19]:
```suggestion
	"status": "provisional",
```

File: wet-boew/tabs/index.json-ld [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+			{
+				"title": "Tabbed Interface",
+				"language": "en",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/tabs/tabs-en.html"
+			},
+			{
+				"title": "Interface à onglets",
+				"language": "fr",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/tabs/tabs-fr.html"
+			}
```
Comment (duboisp) [inline wet-boew/tabs/index.json-ld:31]: Add the Carousel example but listed after those examples.

File: wet-boew/filter/index.json-ld [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+		"examples": [
+			{
+				"title": "Filter",
+				"language": "en",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/filter/filter-en.html"
+			},
+			{
+				"title": "Filtre",
+				"language": "fr",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/filter/filter-en.html"
```
Comment (duboisp) [inline wet-boew/filter/index.json-ld:30]:
```suggestion
				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/filter/filter-fr.html"
```

File: wet-boew/twitter/index.json-ld [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+		"title": {
+		"en": "Twitter",
+		"fr": "Twitter"
```
Comment (duboisp) [inline wet-boew/twitter/index.json-ld:11]: Just a little adjustment to its name as it was renamed for X

```suggestion
		"en": "X (Twitter)",
		"fr": "X (Twitter)"
```

File: wet-boew/wb-randomize/index.json-ld [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+		"docs": [
+			{
+				"title": "Randomize",
+				"language": "en",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/wb-randomize/wb-randomize-en.html"
+			},
+			{
+				"title": "Aléatoire",
+				"language": "fr",
+				"url": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/wb-randomize/wb-randomize-en.html"
```
Comment (duboisp) [inline wet-boew/wb-randomize/index.json-ld:42]: This should point to wet-boew documentation.

**Review by duboisp — CHANGES_REQUESTED** [review]
See the inline comments.

File: wet-boew/eqht-css/index.json-ld (hunk 1 above)
Comment (Garneauma) [reply wet-boew/eqht-css/index.json-ld]: @duboisp Why not have the documentation for the CSS Equalheight and JS Equalheight together? I don't understand why they have to by in a separate section.

**Review by Garneauma — CHANGES_REQUESTED** [review]
Please add, under "page_group" in `_data/i18n.yml` the translation for "other".
en:
  other: Other

fr:
  other: Autre

**Review by Garneauma — APPROVED** [review]

**Pull request merged**